src/ui/public/kuery/node_types/wildcard.js

```diff
@@ -33,3 +33,8 @@ export function toQueryStringQuery(node) {
   const { value } = node;
   return value.split(wildcardSymbol).map(escapeQueryString).join('*');
 }
+
+export function hasLeadingWildcard(node) {
```

Comment:
nit: This seems to be the first function not aimed at building or converting a code, but testing a value. Maybe it makes sense to do this in `buildNode` instead and check for `value. hasLeadingWildcard` in the peg definition?

Reply:
The [test](https://github.com/elastic/kibana/blob/814ae76fde8f05ea1caee77857bf5a3e340ea3ad/src/ui/public/kuery/node_types/wildcard.js#L20-L25) function is also not aimed at building or converting, so I think there's precedence here.

Also, these objects aren't strictly immutable, so the node could change and then `value.hasLeadingWildcard` would be out of date.